cypress.config.ts

```diff
@@ -1,12 +1,18 @@
 import { defineConfig } from 'cypress'
+import dotenv from 'dotenv'
+
+// Load environment variables from .env file
+dotenv.config()
```

Comment:
This needs to have the path as it was not loading any changes to begin with
```suggestion
// Load environment variables from .env file
dotenv.config()
```

Reply:
did this change? I don't see any difference in the suggestion. I'm guessing this is before we talked about the .env stuff

Reply:
This has not changed. I will mark it as reslved @ermish @theyokohamalife